internal/subscriber/subscriber.go

```diff
@@ -157,6 +165,9 @@ func (s *Subscriber) acknowledge(ctx context.Context, req *AckMessage) {
 	}
 
 	removeMessageFromMemory(stats, req.MessageID)
+
+	subscriberMessagesAckd.WithLabelValues(s.topic, s.subscription).Add(1)
+	subscriberTimeTakenToAckMsg.WithLabelValues(s.topic, s.subscription).Observe(time.Now().Sub(msg.PublishTime).Seconds())
```

Comment:
This is diff of the time when the message was published and then acked, right? Should we have a similar metric from publish to consume and from consume to ack/modack?

Reply:
Also, similar metric for modack is missing